src/chain.h

```diff
@@ -380,7 +380,7 @@ class CDiskBlockIndex : public CBlockIndex
 
         // Only read/write nSproutValue if the client version used to create
         // this index was storing them.
-        if ((nType & SER_DISK) && (nVersion >= SPROUT_VALUE_VERSION)) {
```

Comment:
the previous commit has errors `nType` not defined here and doesn't compile.
This change might be moved to the previous commit

Reply:
Ahh OK, this line wasn't in the bitcoin code, that's why it's still wrong here.